cache/cache.cpp

```diff
@@ -281,10 +603,10 @@ int dnet_cmd_cache_lookup(struct dnet_backend_io *backend, struct dnet_net_state
 		return -ENOTSUP;
 	}
 
-	cache_manager *cache = (cache_manager *)backend->cache;
+	auto cache = reinterpret_cast<cache_manager *>(backend->cache);
```

Comment:
static_cast can convert from void* to any pointer type. In this case, it guarantees that if the void* value was obtained by converting from that same pointer type, the resulting pointer value is the same.